src/grpc-method.js

```diff
@@ -78,18 +78,19 @@ class GrpcMethod {
   /**
    * Format errors for consumption by external grpc clients
    *
-   * @param  {error} err Error to be formatted for public consumption
+   * @param  {error}  err        Error to be formatted for public consumption
+   * @param  {Number} statusCode grpc Status code
    * @return {GrpcError}
    */
-  grpcError (err) {
+  grpcError (err, statusCode = grpc.status.INTERNAL) {
```

Comment:
Do you think it makes sense to expose grpc status codes in the library itself, or just allow ANY grpc status code?